sklearn/metrics/_classification.py

```diff
@@ -2723,6 +2735,12 @@ def log_loss(
 
     # Renormalize
     y_pred_sum = y_pred.sum(axis=1)
+    if not np.isclose(y_pred_sum, 1, rtol=1e-15, atol=5 * eps).all():
+        warnings.warn(
+            "The y_pred values do not sum to one. Starting from 1.3 this"
```

Comment:
@lorentzenchr I just want to clarify wherether this should be 1.4 as well? Since I think we might to keep the warning in 1.3 and then raise the error in 1.4?